documentation/zathras_doc.adoc

```diff
@@ -215,6 +215,7 @@ General options
   --no_spot_recover: Do not recover from a spot system going away.
   --package_name &lt;name&gt;: Use this set of packages to override the default in the test config
     file instead of the default. Default format  package name &lt;os&gt;_pkg, new name &lt;os&gt;_pkg_&lt;ver&gt;.
+  --pbench_install_stats: level of pbench stats to insall: light, medium, legacy, heavy.  Default is medium.
```

Comment:
Add a link to documentation regarding what level contains what tools.  It could just be a pointer off the relevant pbench docs.